guide/C/ch_invest.xml

```diff
@@ -1813,6 +1813,840 @@ Income
+              recorded as 2 transactions:
+            </para>
+
+            <table>
+              <title>Transaction 1 dealing with value received, reduction of the
+                number of shares and (optionally) commission
+              </title>
+              <tgroup cols="5">
+                <tbody>
+                  <row>
+                    <entry><emphasis>Account</emphasis></entry>
+                    <entry><emphasis>Tot Shares</emphasis></entry>
+                    <entry><emphasis>(Unit) Price</emphasis></entry>
+                    <entry><emphasis>Buy (Debit)</emphasis></entry>
+                    <entry><emphasis>Sell (Credit)</emphasis></entry>
+                  </row>
+                  <row>
+                    <entry>Brokerage or Bank</entry>
+                    <entry></entry>
+                    <entry></entry>
+                    <entry>Debit</entry>
+                   <entry></entry>
+                  </row>
+                  <row>
+                    <entry>Commission (optional)</entry>
+                    <entry></entry>
+                    <entry></entry>
+                    <entry>Debit</entry>
+                    <entry></entry>
+                  </row>
+                  <row>
+                    <entry>Security</entry>
+                    <entry>-NumSold</entry>
+                    <entry>SaleUnitPrice</entry>
+                    <entry></entry>
+                    <entry>SaleValue</entry>
+                  </row>
+                </tbody>
+              </tgroup>
+            </table>
+
+            <table>
+              <title>Transaction 2 capital gain/loss (loss in this example)
+              </title>
+              <tgroup cols="5">
+                <tbody>
+                  <row>
+                    <entry><emphasis>Account</emphasis></entry>
+                    <entry><emphasis>Tot Shares</emphasis></entry>
+                    <entry><emphasis>(Unit) Price</emphasis></entry>
+                    <entry><emphasis>Buy (Debit)</emphasis></entry>
+                    <entry><emphasis>Sell (Credit)</emphasis></entry>
+                  </row>
+                  <row>
+                      <entry>Capital Loss</entry>
+                      <entry></entry>
+                      <entry></entry>
+                      <entry>Debit</entry>
+                      <entry></entry>
+                  </row>
+                  <row>
+                      <entry>Security</entry>
+                      <entry>0</entry>
+                      <entry>0</entry>
+                      <entry></entry>
+                      <entry>Credit</entry>
+                  </row>
+                </tbody>
+              </tgroup>
+            </table>
+          </listitem>
+
+          <listitem>
+            <para>It has been reported scrubbing does not work correctly if
+              the Stock Split Assistant has been used.
+            </para>
```

Comment:
I'll ask Mike to have a look at this PR. Maybe he'll have some insight into how to document it.